src/js/jsx/Properties.jsx

```diff
@@ -115,6 +116,10 @@ define(function (require, exports, module) {
                     <TransformPanel
                         disabled={disabled}
                         document={document} />
+                    <LibrariesPanel
```

Comment:
I might go so far as to ask that we find a way to disable this stuff by default unless you set a preference? Because, like, we might want to release the next sprint build on the pre-release forum, and although I want to get this merged so that we can get more eyes on it, I also don't want those prerelease builds to be completely broken.